pkg/api/types.go

```diff
@@ -21,7 +21,7 @@ const (
 
 // invalidFilenameCharacters contains a list of character we consider malicious
 // when injecting the directories into containers.
-const invalidFilenameCharacters = `\:;*?"<>|%#$!+{}&[],"'` + "`"
+const invalidFilenameCharacters = `\;*?"<>|%#$!+{}&[],"'` + "`"
```

Comment:
I don't think this is important enough to be concerned about.  IsInvalidFilename is used in one place and frankly IMO is no more than superstition.  It could be ripped out entirely, although perhaps if it stays as it is, it could contribute protection in an unexpected way one day.  Opening the door to ':' characters could result in a new class of invalid path being passed in, sure, but these will still be picked up eventually as an invalid path just like any other.